### ISSUE TITLE
[innogysmarthome] Rename to livisismarthome

### ISSUE THREAD

**Issue by Novanic** [openhab/openhab-addons#10440]
"Innogy SmartHome" was switched in 2020 to the company Livisi GmbH and it is now called "Livisi SmartHome". Therefore the binding should get renamed from innogysmarthome to livisismarthome including various rename changes (packages, classes, README, etc.).

It seems that there will be new devices in the new future which should get integrated and probably some API changes. Maybe it is a good idea to copy the binding with the new name livisismarthome and mark to old binding as deprecated. That would it make possible to do some changes/cleanups which hasn't to be backward compatible.

**Comment by Novanic** [comment]
@ollie-dev Hi, you are entered as the codeowner. What is your opinion (regarding copying the binding with a new name and marking the current as deprecated)? Thanks :-)

**Comment by ollie-dev** [comment]
Hey @Novanic, to be honest - it's been a while since I actively developed this binding. But as far as I know the Livisi guys mentioned maybe a year ago, that renaming would be appreciated. So I think it is a good idea.

What I don't know about is the mentioned API changes. Regarding that I hope that Livisi even with a newer API version still supports all the "ancient" RWE Smarthome devices, so that there is no real compatibility issue at all.

Long story short: renaming as a new binding seems fine for me.

**Comment by Novanic** [comment]
Hello @ollie-dev ,

I have added you to my repository (see branch "livisilocal") if you would like to take a look at it.

There is already the copied/renamed binding, which uses also the new local API which is currently in development/test and it has a few bug fixes and enhancements. But mainly it is still based on your innogy binding and code. The plan is that we mark the innogy binding as deprecated and integrate a warning in the log that the users should migrate to the livisi binding. Are you fine with the plan (but actually you said that before ;-), thanks for that).

And who should be entered as the binding owner? Should I enter you or me? Both would be fine for me.

Thank you and best regards,

Sven

**Comment by ollie-dev** [comment]
Hi @Novanic,

thanks for the info. It is great to hear Livisi is providing a local API to the community and you adopted it. A local API would have always been the better way - nice it's coming at last.

As I wrote before, it's been a while since I actively developed the RWE/Innogy Smarthome binding. Sounds all fine for me and of course you are the owner of the new binding - no need to mention me there. Kudos to you! :)

However, I'll take a peek into your repo. Curious about the new API...

Thanks for your effort and best regards!
Ollie

**Comment by Novanic** [comment]
Hello @ollie-dev 

Thank you. I will of course still mention you and I would also keep the author JavaDoc tags. The most files were only moved/renamed. ;-)

We have also still a few time to refactor the code. At the moment I'm trying to simplify InnogyBridgeHandler and InnogyDeviceHandler. Is there something you would change / rewrite or is there something which isn't actually required anymore?

And I have a few questions, maybe you can help me with answering the questions. :-)

1. There is code to discover the bridge / SHC via mDNS, see class InnogyBridgeDiscoveryParticipant. But it isn't working for me. There are only other devices which are found and the methods of InnogyBridgeDiscoveryParticipant are only called with other devices but with no SHC. Do you know what the reason could be? Or is it possible that it isn't working anymore (for example since any SHC firmware update)?
2. There is a line "DEBUG.equals(device.getConfig().getName())" within InnogyDeviceHandler. It seems there is a possiblity to set the device in debug mode. How is that possible and is it still relevant / required?

Thank you and best regards,

Sven

**Comment by ollie-dev** [comment]
@Novanic regarding your questions:

1. I am unsure about this. Just yesterday after a restart of OH2 mit the innogy binding I found a log entry like "SMARTHOME06 added to Inbox". So there might be some functionality, but I also remember, that it was not reliably working and not for everybody. 
Regarding the method calls - if you are just looking for code references, this might be misleading. It was years ago since I worked on this, but I think there was one XML file that contained a reference to the discovery methods. IIRC, that XML was removed by the migration to karaf. However - long ago and I might be wrong.

2. I do not remember. Looks like a DEBUG switch and I would say, it can be removed.
